enforcer/src/main/java/org/wso2/micro/gateway/enforcer/security/jwt/validator/RevokedJWTDataHolder.java

```diff
@@ -33,6 +34,11 @@
     private static Map<String, Long> revokedJWTMap = new ConcurrentHashMap<>();
     private static RevokedJWTDataHolder instance = new RevokedJWTDataHolder();
 
+    public void init() {
```

Comment:
Need to check event hub enabled 
`f (ConfigHolder.getInstance().getConfig().getEventHub().isEnabled()) {`

Reply:
Fixed with 82a6b47